rosbag2_interfaces/srv/IsPaused.srv

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
why not using http://docs.ros.org/en/api/std_srvs/html/srv/Trigger.html

Reply:
My thinking on ROS best-practices was that it's generally better to define a semantically-specific interface, even if a `std_msgs`/`std_srvs` has the same fields. I can't remember where I first got this idea, though

Reply:
Reasoning something like "`rosbag2_interfaces/SetRate` is self-documenting, but `generic_srvs/SetFloat` doesn't tell you anything about what the service is for".

And in this case, `Trigger` semantically means "make something happen" - which could be confusing considering it's really a "GetBool" operation.

Reply:
I can see a counterpoint being - "if you use `std_srvs`, then you don't introduce a dependency on `rosbag2` in order to have a remote-controller". I think here the problem is that `std_srvs` is so limited, that you still need a custom service for e.g. `Seek` and `GetRate` - so given that the dependency exists anyways, might as well have custom definitions.